Podcast/BookmarkViewController.swift

```diff
@@ -154,6 +163,11 @@ class BookmarkViewController: ViewController, UITableViewDelegate, UITableViewDa
         showActionSheetViewController(actionSheetViewController: actionSheetViewController)
     }
     
+    func didPressActionItemButton() {
```

Comment:
What action button brings up bookmarks?

Reply:
It's for the empty state view! 
<img width="366" alt="screen shot 2017-10-25 at 12 51 04 am" src="https://user-images.githubusercontent.com/14966713/31981054-97537a52-b91e-11e7-85b2-fd8d167bd529.png">